src/app/games/coerceo/CoerceoTutorial.ts

```diff
@@ -9,35 +9,38 @@ const N: number = FourStatePiece.NONE.value;
 const O: number = FourStatePiece.ZERO.value;
 const X: number = FourStatePiece.ONE.value;
 
-export const coerceoDidacticial: DidacticialStep[] = [
+export const coerceoTutorial: DidacticialStep[] = [
     DidacticialStep.informational(
-        $localize`Plateau et but du jeu`,
-        $localize`Le Coerceo se joue sur un plateau comme ceci, composé de tuiles hexagonales, comportant chacune 6 triangles.
-         Les triangles sont les cases où les pièces se déplacent tout le long de la partie.
-         Les tuiles sont séparable du reste du plateau (vous verez comment plus tard).
-         Les pièces foncées appartiennent au premier joueur et ne se déplaceront toute la partie que sur les cases foncées,
-         les pièces claire appartiennent au second joueur et ne se déplaceront également que sur les cases claires.
-         Le but du jeu au Coerceo est de capturer toutes les pièces de l'adversaire.`,
+        $localize`Board and goal of the game`,
+        $localize`Coerceo is played on a board like this, composed of hexagonal tiles, each comprising 6 triangles.
+         The triangles are the spaces along which pieces move during the game.
+         The tiles can be removed from the board (you will see how later).
+         The dark pieces belong to the first player and can only move on the dark spaces,
+         while the light pieces belong to the second player and can only move on the light spaces.
+         The goal of the game is to capture all of the opponent's pieces.`,
         CoerceoPartSlice.getInitialSlice(),
     ),
     DidacticialStep.anyMove(
-        $localize`Deplacement`,
-        $localize`Pour effectuer un déplacement, il faut:
-        1. Cliquer sur l'une de vos pièces.
-        2. Cliquer sur l'une des cases triangulaires encadrées en jaune.
-        Note, vous pouvez passer à travers les pièces adverse.
-        Vous jouez en premier, vous jouez donc Foncé, faites n'importe quel déplacement.
-        Note: peut importe ce que vous faites, aucunes de vos pièces ne peuvent être capturée pendant votre tour.`,
+        $localize`Move`,
+        $localize`To move a piece, you need to:
+        <ol>
+          <li>Click on one of your pieces.</li>
+          <li>Click on one of the triangular space highlighted in yellow.</li>
+        </ol>
+        You can pass through the opponent's pieces.
+        You're playing first, hence you're playing Dark.<br/><br/>
+        Perform any move.
+        Note: no matter what you do, no piece can be captured during your first turn.`,
```

Comment:
none of **your** piece can be captured during your turn.
Et pas durant votre premier tour, c'est quand c'est ton tour même si tu provoque la disparition de ta dernière liberté, ta pièce reste !

Reply:
Est-ce que cette note est utile ici ? On est a l'étape 2 du tuto, le joueur apprends à bouger une pièce et on lui dit « ah tiens tu vois ce concept de liberté qu'on t'as pas encore expliqué, idem pour la capture ? Ben c'est bon en fait si y'a toujours des libertés y'a pas de capture »

Reply:
Oui allez, c'est expliqué dans special-capture c'est bien assez, tu peux supprimer ça des deux fichiers \o/